Solutions/Web Session Essentials/Hunting Queries/RequestFromBotsAndCrawlers.yaml

```diff
@@ -0,0 +1,41 @@
+id: b653c2e0-ae0f-490b-9b2e-836858d28976
+name: Request from bots and crawlers (ASIM Web Session)
+description: |
+  'This query searches for requests from bots and crawlers.'
+tags:
+  - Schema: WebSession
+    SchemaVersion: 0.2.6
+requiredDataConnectors: []
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  let lookback =- 1d;
```

Comment:
Looks like we have an extra character here (-).

Reply:
fixed